### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Erchu420**
- [DEMO LINK](https://Erchu420.github.io/layout_stars/)
- [TEST REPORT LINK](https://Erchu420.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,7 +7,50 @@
...
+  <body class="page">
+    <div class="block">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
```
Comment (asakevych) [inline src/index.html:19]: add empty lines between stars blocks

**Review by asakevych — CHANGES_REQUESTED** [review]
Great job!! perfect implementation!! just small change to prettier html is needed 😉

File: src/index.html (hunk 1 above)
Comment (Erchu420) [reply src/index.html]: added in solution2 ;)

File: src/index.html [hunk 2]
```diff
@@ -7,7 +7,55 @@
...
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <div class="block">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
```
Comment (choeqq) [inline src/index.html:17]: you can use self-closing divs `<div />` when there is no children

**Review by choeqq — APPROVED** [review]
Smart solution 👍

**Pull request closed without merging**